### DISCUSSION TITLE
Disabling Telemetry

### DISCUSSION THREAD

**Discussion by ghost** [HotCakeX/Harden-Windows-Security#462] [Q&A]
Do you have a recommendation for disabling telemetry?

**Comment by HotCakeX** [comment]
Hi,
Just added it to the roadmap
https://github.com/HotCakeX/Harden-Windows-Security/issues/479

**Comment by mketab** [comment]
Something good to know is that completely disabling diagnostic data disables Smart App Control without any prompt. So, there will always be some data being collected by Microsoft if you want all security features enabled.

**Reply by HotCakeX** [reply]
That's true. telemetry is not a bad thing. in the enterprise world, companies install something called EDR (Endpoint Detection and Response) among many other things or have contracts with SoCs (security operations center).

To put it simply, they literally give their data to others in order to stay protected. That's how things work. One should choose between which party will have one's data, either let friendlies have access to telemetry so they can protect you, or let ransomware and malware infect your device and steal the data. The choice is clear to me.

Now what got end/home users so worried about telemetry in Windows? Lots of disinformation campaigns. I see it all the time, whether it's YouTube video, a post on X, news websites etc. doesn't matter what agenda they have, either looking for clicks to generate revenue, looking for more views again for money, or they like one OS and want to campaign against the other by spreading lies.

The point is that we shouldn't fall for them and have to think clearly.

**Reply by ghost** [reply]
I understand your perspective. However, since those options are available to be modified in GPE, it might not be detrimental for the end user to have that choice.

**Comment by HotCakeX** [comment]
Time to wrap this up because it's completed, implemented 14 policies as a sub-category for the Miscellaneous category. More info here: https://github.com/HotCakeX/Harden-Windows-Security/pull/510 and here https://github.com/HotCakeX/Harden-Windows-Security/issues/479#issuecomment-2575031661
